sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -490,7 +490,7 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
     }
   }
 
-  test("Fiters should be pushed down for vectorized Parquet reader at row group level") {
+  ignore("Fiters should be pushed down for vectorized Parquet reader at row group level") {
```

Comment:
since we do not send accumulator names to executor, this test is not valid anymore. cc @viirya is there any other ways to test it?